base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp

```diff
@@ -612,18 +612,16 @@ Result<cvd::Response> CvdStartCommandHandler::Handle(
                                .follow_symlink = false}));
     }
   }
-  CF_EXPECT(Contains(envs, kAndroidHostOut));
-  const auto bin = CF_EXPECT(FindStartBin(envs.at(kAndroidHostOut)));
-
   // update DB if not help
   // collect group creation infos
   CF_EXPECT(Contains(supported_commands_, subcmd),
             "subcmd should be start but is " << subcmd);
   const bool is_help = CF_EXPECT(IsHelpSubcmd(subcmd_args));
-  CF_EXPECT(ConsumeDaemonModeFlag(subcmd_args));
-  subcmd_args.push_back("--daemon=true");
 
   if (is_help) {
+    CF_EXPECT(Contains(envs, kAndroidHostOut));
+    const auto bin = CF_EXPECT(FindStartBin(envs.at(kAndroidHostOut)));
```

Comment:
`at` can throw an exception and `Contains` + `at` is a double lookup. Can this use `find`?

See [go/totw/132#safe-lookup](https://go.corp.google.com/totw/132#safe-lookup)

Reply:
Done